gnomad_qc/v4/annotations/generate_variant_qc_annotations.py

```diff
@@ -519,6 +733,55 @@ def main(args):
         action="store_true",
     )
 
+    variant_qc_annotation_args = parser.add_argument_group(
+        "Variant QC annotation HT parameters"
+    )
+    variant_qc_annotation_args.add_argument(
+        "--create-variant-qc-annotation-ht",
+        help="Creates an annotated HT with features for variant QC.",
+        action="store_true",
+    )
+    variant_qc_annotation_args.add_argument(
+        "--impute-features",
+        help="If set, imputation is performed for variant QC features.",
+        action="store_true",
+    )
+    variant_qc_annotation_args.add_argument(
+        "--n-partitions",
+        help="Desired number of partitions for variant QC annotation HT .",
+        type=int,
+        default=5000,
+    )
+
+    tp_vcf_args = parser.add_argument_group(
+        "Export true positive VCFs",
+        "Arguments used to define true positive variant set.",
+    )
+    tp_vcf_args.add_argument(
+        "--export-true-positive-vcfs",
+        help=(
+            "Exports true positive variants (--transmitted-singletons and/or"
+            " --sibling-singletons) to VCF files."
+        ),
+        action="store_true",
+    )
```

Comment:
Is there ever a use case where either --transmitted-singletons and/or --sibling-singletons are passed but not --export-true-positives-vcfs ? Do they, or the PipelineStepResourceCollection named 'export_true_positive_vcfs', have any other function in the code or are read in elsewhere ? 

If there isn't, you could just choose to export if either true positive types are passed - it would cut down on arguments but might be less user-friendly when passing argument. 

Think the only code change would be changing line #668 to `if true_positive_type:` , but only if this is the case